src/components/Profile/index.js

```diff
@@ -1,37 +1,55 @@
 import React, { useState } from 'react'
 import './styles.scss'
+import propTypes from 'prop-types'
 import ProfileImage from './ProfileImage'
 import Description from './Description'
 import Interests from './Interests'
 import EditButton from './EditButton'
 import ProfileHeader from './ProfileHeader'
 import DescriptionTextEdit from './DescriptionTextEdit'
+import Instructions from './Instructions'
 
 const Profile = props => {
-  const { user } = props || {}
+  const { user, currentUser } = props
   const [editProfile, setEditProfile] = useState(false)
+  const [showModal1, setShowModal1] = useState(true)
+  const [showModal2, setShowModal2] = useState(true)
   const descriptionText = 'Esimerkkikuvaus käyttäjästä'
 
   const toggleEditProfile = () => setEditProfile(!editProfile)
+  const closeModal = i => () =>
```

Comment:
Modal 1 and Modal 2 don't really tell anything. Maybe use enums? 

`MODAL_THIS_IS_YOUR_PROFILE = 1`
`MODAL_PROFILE_INFORMATION_IS_PUBLIC = 2`